R/generate_simulated_data.R

```diff
@@ -100,7 +104,9 @@
 #'   ),
 #'   phi_rt = 0.6,
 #'   sigma_eps = sqrt(0.02),
+#'   spatial_deviation_init = c(0.09, 0.32, 0.09, -0.78, -0.24, 0.16),
```

Comment:
I might suggest, since these should be random variables, that you actually draw this and the `state_deviation_init` within this function rather than passing them in